### ISSUE TITLE
Build scripts seems to be outdated

### ISSUE THREAD

**Issue by aegis-dev** [MasterQ32/SDL.zig#165]
Hello! I am trying to get into Zig language and I was trying to use your project as dependency. Unfortunately, I am encountering lots of error in SDL.zig's `build.zig`. I fixed issue related to `@fieldParentPtr("step", step);` but other issues coming out. I am not qualified to fix your build systems so sorry... no PRs. However, I am willing to help reproducing the issues. 

I am on Windows 11, installed latest Zig which is at the moment `0.11.0`.  

Let me know if I can provide anything else.

**Comment by MasterQ32** [comment]
Yes, please provide a diff of your changes, easiest way via PR. Enable contributor access, so others can continue with the work

**Comment by aleloi** [comment]
I've noticed the same thing. I made a PR that changes the build script in README.md that solves it for me on Zig 0.13. https://github.com/ikskuh/SDL.zig/pull/192
